rio_viz/app.py

```diff
@@ -145,7 +144,7 @@ def __attrs_post_init__(self):
         self.register_middleware()
         self.register_routes()
         self.app.include_router(self.router)
-        self.app.mount("/static", StaticFiles(directory=src_dir), name="static")
```

Comment:
I've had a lot of issue with static files (usually some race condition) so decided to revert